tests/10apidoc/30room-create.pl

```diff
@@ -16,7 +16,7 @@
          content => {
             visibility      => "public",
             # This is just the localpart
-            room_alias_name => "30room-create",
+            room_alias_name => "30room-create-".time,
```

Comment:
Why was this added?

Reply:
it was causing test fails when run on an existing db if that alias already existed.